src/role.test.ts

```diff
@@ -0,0 +1,5 @@
+import { Role } from './role';
+
+it('Role enum contains Standard', () => {
+	expect(Role.Standard).toBeDefined();
+});
```

Comment:
Similar to https://github.com/guardian/libs/pull/220#discussion_r657176486, I'm not sure if this is useful beyond keeping coveralls happy.

Reply:
At the very least it means that if you’re changing a type, you need to change it in two places, which seems like it’s a good reminded that it is a breaking change?